src/transport/tls/config.rs

```diff
@@ -506,6 +521,39 @@ mod test_util {
                 private_key: dir.join(self.private_key),
             }
         }
+
+        // Returns a `ConnectionConfig<ClientConfigWatch>` preloaded with a
+        // valid client TLS configuration.
+        pub fn client(&self, server_identity: Identity) -> (ConnectionConfig<ClientConfigWatch>)
```

Comment:
Do the parentheses here indicate that this returned inside of a 1-element tuple? Is there a reason for this?